README.md

```diff
@@ -1,3 +1,7 @@
+<p align="center">
```

Comment:
Since this is just a container consider using `<div align="center">`

Reply:
Done.